pkg/manifests/manifests_test.go

```diff
@@ -1034,6 +1034,7 @@ func TestPrometheusK8sRemoteWrite(t *testing.T) {
 func TestPrometheusK8sConfiguration(t *testing.T) {
 	c, err := NewConfigFromString(`prometheusK8s:
   retention: 25h
+  retentionSize: 10Gi
```

Comment:
`10Gi` isn't a valid value.

```suggestion
  retentionSize: 10GB
```

Reply:
sure I updated it to 10GiB I missed it earlier